runtime/vm/UpcallThunkMem.cpp

```diff
@@ -27,6 +27,8 @@
 
 extern "C" {
 
+#define ROUND_TO(granularity, number) (((UDATA)(number) + (granularity) - 1) & ~((UDATA)(granularity) - 1))
```

Comment:
we have a few of these, maybe we can just define one in j9.h and add a c++ version in VMHelpers. Can be done in a separate PR

Reply:
Actually, we can use OMR/Bytes.hpp::align here

Reply:
Updated to use `OMR/Bytes.hpp::align`.